### PR TITLE
feat: small change to schizophrenia diagnosis, removed redundant diag…

### PR DESCRIPTION
**Pull request by erikperfalk**
…noses in misc psychosis,added underscores

- [X] I have battle-tested on Overtaci (RMAPPS1279)
- [X] I have bumped the version as described in the [wiki](https://www.notion.so/Setting-up-Poetry-on-your-local-machine-76647d75e8fa4d9ba553cbc2a49946d9#dca1e0f2ce934a49acd1851aaf882d75)

## Notes for reviewers
feat: small change to schizophrenia diagnosis, removed redundant diagnoses in misc psychosis,added underscores

### CODE REVIEW

File: src/psycopmlutils/loaders/raw/load_diagnoses.py [hunk 1]
```diff
@@ -298,39 +298,39 @@ def f1_disorders(n: Optional[int] = None) -> pd.DataFrame:
...
         )
 
-    @data_loaders.register("cannabinoid and hallucinogen dependencies")
+    @data_loaders.register("cannabinoid_and_hallucinogen_dependencies")
     def cannabinoids_and_hallucinogens(n: Optional[int] = None) -> pd.DataFrame:
         return LoadDiagnoses.from_physical_visits(
             icd_code=["f12", "f16"],
             wildcard_icd_code=True,
             n=n,
         )
```
Comment (MartinBernstorff) [inline src/psycopmlutils/loaders/raw/load_diagnoses.py:323]: Are there good arguments for having these collected in the same columns? I'd recommend splitting them up in the loader functions (e.g. `cannbinoid_dependency` and `hallucinogen_dependency` separately), and then we can choose to join them during pre-processing :-)

**Review by MartinBernstorff — COMMENTED** [review]
Looks good to me! One tiny adjustment, then looks ready to merge.

**Review by MartinBernstorff — APPROVED** [review]

File: src/psycopmlutils/loaders/raw/load_diagnoses.py (hunk 1 above)
Comment (erikperfalk) [reply src/psycopmlutils/loaders/raw/load_diagnoses.py:323]: Not really :) So good point! I'll split them up, commit and then merge

**Comment by MartinBernstorff** [comment]
In the future, try to avoid deleting comments, and instead mark the conversation as resolved :-)

**Comment by erikperfalk** [comment]
Noted. and sorry! pre-commit run messed with me, so I tried everything. Including your comment ;)

**Pull request merged**